orchagent/muxorch.cpp

```diff
@@ -830,9 +836,90 @@ bool MuxOrch::isNeighborActive(IpAddress nbr, string alias)
         return ptr->isActive();
     }
 
+    string port;
+    if (!getMuxPort(mac, alias, port))
+    {
+        SWSS_LOG_INFO("Mux get port from FDB failed for '%s' mac '%s'",
```

Comment:
Q: Is this a true error case or it can happen "normally" during init time or due to in certain mux state?
If it is truly an error case, then this needs to be logged as error.
If it can happens at init time then it is ok as is.
But if this can happen quite regularly and not an error, perhaps we should set the log level to DEBUG so it does not keep filling up the syslog...

Reply:
This can return false if neighbor is learnt over a portchannel interface (T1) for which there is no mux port associated. For such entries, it is not expected to happen frequently.